packages/ses-ava/src/command.js

```diff
@@ -0,0 +1,155 @@
+/* global process */
+/* eslint-disable no-await-in-loop, no-continue, no-labels, no-unreachable-loop */
+
+/* The ses-ava command allows a single package to run the same test suite with
+ * multiple configurations named in package.json under `sesAvaConfigs`.
+ * Each of these configurations requires a separate Ava config file like
+ * test/_ava.special.config.js and a name.
+ * The ses-ava command will by default run all tests in every mode but allows
+ * the user to pass --only-* and --no-* at any argument position for any of
+ * the named configurations to filter.
+ * Consequently, the "test" script for a package using ses-ava can simply be
+ * "ses-ava" and preserves the filtering behaviors of the underlying "ava"
+ * sessions if run like `yarn test -m file`.
+ */
+
+import popen from 'node:child_process';
+import fs from 'node:fs';
+
+const singlePassThrough = new Set([
+  '-t',
+  '--tap',
+  '-s',
+  '--serial',
+  '-u',
+  '--update-snapshots',
+  '--no-worker-threads',
+  '--verbose',
+  '--no-verbose',
+]);
+
+const doublePassThrough = new Set(['-m', '--match', '--node-arguments']);
```

Comment:
Suggestions (formatting short/long aliases on the same line and renaming these two variables):
```suggestion
// We pass these flags directly to ava.
const passThroughFlags = new Set([
  ...['-s', '--serial'],
  ...['-t', '--tap'],
  ...['-T', '--timeout'],
  ...['-u', '--update-snapshots'],
  ...['-v', '--verbose'],
  '--no-worker-threads',
  '--no-verbose',
]);

// We read a single argument for each of these options and pass the pair
// directly to ava.
const passThroughArgOptions = new Set(['-m', '--match', '--node-arguments']);
```